timeseries/src/autogluon/timeseries/dataset/ts_dataframe.py

```diff
@@ -1008,3 +1008,7 @@ def __dir__(self) -> List[str]:
         # This hides method from IPython autocomplete, but not VSCode autocomplete
         deprecated = ["get_reindexed_view", "to_regular_index"]
         return [d for d in super().__dir__() if d not in deprecated]
+
+    def to_data_frame(self) -> pd.DataFrame:
+        """Convert `TimeSeriesDataFrame` to a `pandas.DataFrame`"""
+        return pd.DataFrame(self)
```

Comment:
Do we need to reset the index of the DataFrame?

Reply:
The users should be able to do that themselves afterwards